src/log-locales-missing-keys.js

```diff
@@ -0,0 +1,9 @@
+function logResult(missingKeys) {
```

Comment:
Maybe rename this function to have the same name as the file?

Reply:
Sure! thanks for noticing it!

Reply:
✅